uqcsbot/scripts/yt.py

```diff
@@ -36,16 +36,16 @@ def handle_yt(command: Command):
         bot.post_message(command.channel_id, "Your query returned no results.")
 
 
-def get_top_video_result(search_query: str):
+def get_top_video_result(search_query: str, channel):
     '''
     The normal method for using !yt searches based on query
     and returns the first video result. "I'm feeling lucky"
     '''
     search_response = execute_search(search_query,
```

Comment:
put the comment above
i.e. "Return one video id."

Reply:
(the inline comments make things a bit untidy, and I reckon the explanations are simple enough to chuck it above)